raiden_installer/utils.py

```diff
@@ -75,17 +75,16 @@ def send_raw_transaction(w3, account, contract_function, *args, **kw):
     return w3.eth.waitForTransactionReceipt(tx_hash, timeout=WEB3_TIMEOUT)
 
 
-def wait_for_transaction(web3: Web3, transaction_receipt: Dict[str, Any]) -> None:
-
+def wait_for_transaction(w3: Web3, transaction_receipt: Dict[str, Any]) -> None:
     if "blockNumber" not in transaction_receipt:
         raise KeyError("blockNumber not in transaction receipt.")
 
     block_with_transaction = transaction_receipt["blockNumber"]
-    current_block = web3.eth.blockNumber
+    current_block = w3.eth.blockNumber
 
     while current_block < block_with_transaction:
```

Comment:
```suggestion
    while current_block < block_with_transaction + confirmation_interval:
```

Reply:
While this probably reduces the error chance in case of reorgs, it makes not entirely sure that the transaction is included in a block. In case of reorgs the tx receipt might change. Maybe we should try to get the receipt in every loop iteration? Do you think that makes sense?

Reply:
yes totally! nice catch